### PR TITLE
data: sandramsc

### PR DESCRIPTION
**Pull request by sandramsc**
fix json syntax error at "Web3"

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6113"></a>

### CODE REVIEW

**Comment by sandramsc** [comment]
not sure why the JSON check keeps failing:

I restructured and edited the json multiple times to debug where this error is even by copying and pasting Eddies's json into mine (passed all checks) -- then replacing it with my data which failed 3 more times...

File: data/sandramsc.json [hunk 1]
```diff
@@ -1,41 +1,59 @@
...
+    },
+    {
+      "group": "Socials Primary",
+      "name": "LinkedIn: Let's connect",
       "url": "https://linkedin.com/in/sandraashipala",
-      "icon": "FaLinkedin"
-    }
+      "icon": "FaLinkedin",
+      "color": "#0000FF"
+    },
```
Comment (SaraJaoude) [inline data/sandramsc.json:32]: remove comma
```suggestion
    }
```

**Comment by SaraJaoude** [comment]
> not sure why the JSON check keeps failing:
> 
> I restructured and edited the json multiple times to debug where this error is, even by copying and pasting Eddies's json into mine (passed all checks) -- then replacing it with my data which failed 3 more times...

I copied/pasted the file into jsonlint.com and hopefully the inline comment I have made resolves this.  We should be able to merge your PR once you change the information back to your own name/bio

**Comment by sandramsc** [comment]
> > not sure why the JSON check keeps failing:
> > I restructured and edited the json multiple times to debug where this error is, even by copying and pasting Eddies's json into mine (passed all checks) -- then replacing it with my data which failed 3 more times...
> 
> I copied/pasted the file into jsonlint.com and hopefully the inline comment I have made resolves this. We should be able to merge your PR once you change the information back to your own name/bio

Thanks @SaraJaoude!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**